apache-rat-core/src/main/java/org/apache/rat/Report.java

```diff
@@ -59,74 +65,166 @@
 import org.apache.rat.walker.ArchiveWalker;
 import org.apache.rat.walker.DirectoryWalker;
 
+import static java.lang.String.format;
+
 /**
  * The CLI based configuration object for report generation.
  */
 public class Report {
+
+    private static final String[] NOTES = {
+            "Rat highlights possible issues.",
+            "Rat reports require interpretation.",
+            "Rat often requires some tuning before it runs well against a project.",
+            "Rat relies on heuristics: it may miss issues"
+    };
+
+    private static final String[] STYLE_SHEETS = {"plain-rat", "missing-headers", "unapproved-licenses"};
```

Comment:
Should we extract this into an enum - 
apart from the text/CLI option value it could also contain the location of the corresponding XSLT and a description.

WDYT?